contrib/internal/httptrace/httptrace.go

```diff
@@ -157,7 +160,6 @@ func isGlobal(ip netaddr.IP) bool {
 
 // urlFromRequest returns the full URL from the HTTP request. If query params are collected, they are obfuscated granted
 // obfuscation is not disabled by the user (through DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP)
-// For more information see https://datadoghq.atlassian.net/wiki/spaces/APM/pages/2357395856/Span+attributes#http.url
```

Comment:
Same here, a link to https://docs.datadoghq.com/tracing/configure_data_security#redacting-the-query-in-the-url wouldn't hurt.